packages/suite/src/views/wallet/trading/common/TradingContainer.tsx

```diff
@@ -10,13 +15,17 @@ export interface TradingContainerProps {
 
 export const TradingContainer = ({ SectionComponent }: TradingContainerProps) => {
     const selectedAccount = useSelector(state => state.wallet.selectedAccount);
+    const isDiscoveryRunning = useSelector(selectHasRunningDiscovery);
 
     if (selectedAccount.status !== 'loaded') {
         return <TradingLayoutHeader />;
     }
 
     return (
         <TradingLayoutHeader>
+            <Column margin={{ bottom: spacings.md }}>
+                {isDiscoveryRunning && <DiscoveryWarning />}
+            </Column>
```

Comment:
When discovery is complete, I wouldn't render the `Column` component.
```suggestion
            {isDiscoveryRunning && (
                <Column margin={{ bottom: spacings.md }}>
                    <DiscoveryWarning />
                </Column>
            )}
```

Reply:
fixed in https://github.com/trezor/trezor-suite/pull/20221/commits/eced72437d74743a7052b0afaecec6cb7e26ba7a